apitesters/purchases.ts

```diff
@@ -157,3 +157,13 @@ async function checkBeginRefundRequest(entitlementInfo: PurchasesEntitlementInfo
   const refundStatus2: REFUND_REQUEST_STATUS = await Purchases.beginRefundRequestForEntitlement(entitlementInfo);
   const refundStatus3: REFUND_REQUEST_STATUS = await Purchases.beginRefundRequestForProduct(storeProduct);
 }
+
+
+async function checkSyncObserverModeAmazonPurchase(productID: string,
+                                                   receiptID: string,
+                                                   amazonUserID: string,
+                                                   isoCurrencyCode?: string | null,
+                                                   price?: number | null): Promise<void> {
+  return Purchases.syncObserverModeAmazonPurchase(
+    productID, receiptID, amazonUserID, isoCurrencyCode, price);
```

Comment:
Should we also api test not passing currency code and price?

Reply:
Actually, since the method also has optional parameters I think that should be covered 👍